civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.rules_defaults.inc

```diff
@@ -22,8 +22,8 @@ function civihr_employee_portal_features_default_rules_configuration() {
             "active" : "1",
             "from_mail" : "[site:mail]",
             "subject" : "Employee Access Request",
-            "body" : "User [current-user:contact_display_name] with username: [current-user:name] ([current-user:mail]) has requested access to your CiviHR Self Service Portal.\\r\\n<a href = \'[site:url]admin/people\'>Please login to activate this account and assign appropriate access level</a>.",
-            "plaintext" : "User [current-user:contact_display_name] with username: [current-user:name] ([current-user:mail]) has requested access to your CiviHR Self Service Portal.\\r\\n<a href = \'[site:url]admin/people\'>Please login to activate this account and assign appropriate access level</a>.",
+            "body" : "User [current-user:contact_display_name] with username: [current-user:name] ([current-user:mail]) has requested access to your CiviHR Self Service Portal.\\r\\n\\u003Ca href = \\u0027[site:url]admin\\/people\\u0027\\u003EPlease login to activate this account and assign appropriate access level\\u003C\\/a\\u003E.",
+            "plaintext" : "User [current-user:contact_display_name] with username: [current-user:name] ([current-user:mail]) has requested access to your CiviHR Self Service Portal.\\r\\n\\u003Ca href = \\u0027[site:url]admin\\/people\\u0027\\u003EPlease login to activate this account and assign appropriate access level\\u003C\\/a\\u003E.",
```

Comment:
we need to check why this has been updated too

Reply:
Not sure why, but it seems your commit was the one that introduced the change:
https://github.com/compucorp/civihr-employee-portal/blame/24340763e2cb9046add4e544b4cb7da1b9d4fe42/civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.rules_defaults.inc#L25

Should I revert just this bit or leave it as it is?

Reply:
No need to revert. The ticket description has been updated with more details about this